src/requests/features.jl

```diff
@@ -183,23 +183,26 @@ function format_text(text::AbstractString, params, config)
     end
 end
 
-function mark_range(text::AbstractString, startline, stopline, startmark, stopmark)
-    textlines = split(text, "\n")
-    stopline_size = sizeof(textlines[stopline])
-    insert!(textlines, stopline + 1, stopmark)
-    insert!(textlines, startline, startmark)
-    return join(textlines, "\n"), stopline_size
-end
+# Broken up to make this file formattable
```

Comment:
Not entirely sure what you mean by this.

Reply:
The two constant strings below are broken up and joined with `*` so that the marker doesn't exist in this file :) I will clarify the comment.